modules/javafx.graphics/src/main/native-glass/mac/GlassWindow+Java.m

```diff
@@ -212,34 +212,30 @@ - (void)_grabFocus
     s_grabWindow = window;
 }
 
-- (void)_setResizable
+- (void)_setResizable:(bool)resizable
```

Comment:
Good. It definitely seems cleaner to pass in resizable as a parameter rather than having this method be a "toggle" resizable as it was previously.